src/IO/Importers/Tags.hpp

```diff
@@ -68,15 +69,19 @@ struct ObservationValueEpsilon {
 /*!
  * \brief Toggle interpolation of numeric data to the target domain
  */
-struct EnableInterpolation {
-  static std::string name() { return "Interpolate"; }
+struct ElementsMatch {
```

Comment:
This name is a bit obscure and I'm not sure users will have a sense of what this option does just from the name. A good name doesn't immediately stand out to me (and I see why you removed the word "interpolate" from the name since we can still do interpolation for matching grids). Here are some ideas but I'm open to more
- `InterpolateAcrossBlocks{Elements}` (or the opposite `InterpolateWithinElements`)
- `IdenticalVolumeFileDomain` ( `IdenticalH5Domain`, `IdenticalSerializedDomain`)
- `IdenticalDomains` ( `IdenticalDomainStructure`)

Reply:
changed to `ElementsAreIdentical`